packages/manager/apps/telecom/src/app/telecom/pack/migration/pack-migration.less

```diff
@@ -76,4 +77,96 @@
   .subservice-title {
     font-size: 1.5em !important;
   }
+
+  .card {
+    position: relative;
+    display: flex;
+    flex-direction: column;
+    min-width: 0;
+    word-wrap: break-word;
+    background-color: #fff;
+    background-clip: border-box;
+    border: 1px solid rgba(0, 0, 0, 0.125);
+    border-radius: 0.25rem;
+    margin: 15px;
+
+    .meeting {
+      height: 250px;
+    }
+  }
+
+  .card-body {
+    flex: 1 1 auto;
+    padding: 1.25rem;
+  }
+
+  .card-title {
+    margin-bottom: 0.75rem;
+  }
+
+  .text-small {
+    font-size: 80%;
```

Comment:
**nitpick:** Avoid using custom css. Please only use the utility classes available from Bootstrap.